### PR TITLE
Set up Dependabot

### PR DESCRIPTION
**Pull request by mmorel-35**

### CODE REVIEW

**Comment by kolyshkin** [comment]
AFAIK these days there's no need to have an explicit dependabot config; dependabot can be set up by repo admin via github UI by visiting "Insights" tab, choosing "Dependency graphs" and then "Dependabot" tab (the direct URL should be https://github.com/cpuguy83/go-md2man/network/updates or so).

**Comment by mmorel-35** [comment]
Hi @kolyshkin,  is it possible to configure the equivalent of the content this file (package-ecosystems, update frequency, pr limit...) though this tab? I'm very interested to now more about this, if you can help

**Comment by cpuguy83** [comment]
The option is there, but it still wants a config for it, so this PR seems to be needed.

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+version: 2
+updates:
+- package-ecosystem: gomod
+  directory: /
+  schedule:
+    interval: daily
+- package-ecosystem: github-actions
+  directory: /
+  schedule:
+    interval: daily
```
Comment (cpuguy83) [inline .github/dependabot.yml:10]: I think weekly ought to be enough for both of these... maybe even monthly.

Comment (mmorel-35) [reply .github/dependabot.yml:10]:
```suggestion
    interval: weekly
```

File: .github/dependabot.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+version: 2
+updates:
+- package-ecosystem: gomod
+  directory: /
+  schedule:
+    interval: daily
```
Comment (mmorel-35) [inline .github/dependabot.yml:6]:
```suggestion
    interval: weekly
```

**Pull request merged**